src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -121,6 +122,12 @@ public void robotInit() {
 		dashboard = new MatchDashboard(subsystems);
 
 		RobotController.setBrownoutVoltage(5.75);
+
+		CommandScheduler.getInstance()
+				.schedule(
+						new SyncPivotRelativeEncoderCommand(subsystems.launcherSubsystem)
+								.ignoringDisable(true)
+								.withName("Sync Relative Encoder"));
```

Comment:
This is only meant to be done once shortly after robot code startup, right?

Reply:
yes